xarray/tutorial.py

```diff
@@ -77,9 +82,26 @@ def load_dataset(name, cache=True, cache_dir=_default_cache_dir,
             """
             raise IOError(msg)
 
-    ds = _open_dataset(localfile, **kws).load()
+    ds = _open_dataset(localfile, **kws)
 
     if not cache:
         _os.remove(localfile)
 
     return ds
+
+
+def load_dataset(*args, **kwargs):
+    """
+    `load_dataset` will be removed in version 0.12. The current behavior of
+    this function can be achived by using `tutorial.open_dataset(...).load()`.
+
+    See Also
+    --------
+    open_dataset
+    """
+    warnings.warn(
+        "load_dataset` will be removed in version 0.12. The current behavior "
+        "of this function can be achived by using "
+        "`tutorial.open_dataset(...).load()`.",
+        FutureWarning, stacklevel=2)
```

Comment:
Maybe `DeprecationWarning`? Generally we save `FutureWarning` for things that are actually going to break in the future due to having different behavior.